### PR TITLE
Feature/search

### PR DESCRIPTION
**Pull request by joanafbrito**
# Description drawing

Created and Add Logic to Search Bar component

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactor feature (non-breaking change that modifies existing work)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

- [x] local host 3000
- [x] console.log()
- [x] dev tools
- [ ] cypress testing

# Checklist:

- [x] My code follows the style guidelines of this project/linted
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/Search/Search.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+
+
+
+class Search  extends Component  {
```
Comment (hayleyw7) [inline src/components/Search/Search.js:5]: Great start! I dig how we got this started.

File: src/components/Search/Search.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        this.state = {
+            foundPokemon:''
+        }
+    }
+
+    handleChange = (e) => {
+        this.setState({[e.target.name]: e.target.value})
+    }
+
+    handleClick = (e) => {
```
Comment (hayleyw7) [inline src/components/Search/Search.js:17]: It works!

File: src/components/Search/Search.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        const foundPokemon = {
+            id:Date.now(),
+            ...this.state
+        }
+        console.log(foundPokemon)
+        // foundPokemon will be = data.name 
+    }
+
+    render() {
+        return (
```
Comment (hayleyw7) [inline src/components/Search/Search.js:28]: Functional & ready for the next logic steps & styling.

**Review by hayleyw7 — APPROVED** [review]

File: src/components/Search/Search.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+                Catch
+                </button>
+            </form>
+        )
+
+}
+
+}
+
+export default Search;
```
Comment (Matt-Roden) [inline src/components/Search/Search.js:48]: This is a nice and simple Search, I like it.

File: src/components/Search/Search.js (hunk 3 above)
Comment (Matt-Roden) [reply src/components/Search/Search.js:28]: 👍

File: src/components/Search/Search.js [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+    }
+
+    handleChange = (e) => {
+        this.setState({[e.target.name]: e.target.value})
+    }
+
+    handleClick = (e) => {
+        e.preventDefault();
+        const foundPokemon = {
+            id:Date.now(),
```
Comment (Matt-Roden) [inline src/components/Search/Search.js:20]: ID's are helpful down the line, good choice to include one.

**Review by Matt-Roden — APPROVED** [review]

**Pull request merged**